changelogs/fragments/default-test-container160.yml

```diff
@@ -1,2 +1,2 @@
 minor_changes:
-  - ansible-test - default container now uses default-test-container 2.7.0 and ansible-base-test-container 1.6.0. This brings in Python 3.9.0rc1 for testing.
+  - ansible-test - default container now uses default-test-container 2.7.0 and ansible-core-test-container 1.6.0. This brings in Python 3.9.0rc1 for testing.
```

Comment:
Guessing these changelog changes for ansible-base-test-container should be reverted

Reply:
I kept it on the premise that we're going to eventually rename the container anyway, but I can revert it for now.

Reply:
Done.